owid/catalog/catalogs.py

```diff
@@ -391,6 +391,24 @@ def find_one(*args: Optional[str], **kwargs: Optional[str]) -> Table:
     return find(*args, **kwargs).load()  # type: ignore
 
 
+def find_latest(
+    table: Optional[str] = None,
+    namespace: Optional[str] = None,
+    dataset: Optional[str] = None,
+    channels: Iterable[CHANNEL] = ("garden",),
+) -> Table:
+    global REMOTE_CATALOG
```

Comment:
We could wrap all this into `REMOTE_CATALOG = _load_remote_catalog(channels)` and reuse it in both functions.

Reply:
True, done!